config/routes.rb

```diff
@@ -16,7 +16,8 @@
   resources :products, except: [:destroy] do
     resources :reviews, only: [:new, :create]
   end
-  patch "/products/retire", to: "products#retire", as: "retire"
+
+  post "/products/:id/retire", to: "products#retire", as: "retire"
```

Comment:
I think this route change makes sense! Would love for you to explain the switch from patch to post to me (I trust you that that's right, just want to understand the route better)

Reply:
The 'patch' to 'post' was tricky! I had it originally as patch, but it couldn't find the route. When I switched it to post, that worked. I had to do it that way in my tasklist also

Reply:
I changed it to patch and it still works! Resolving this comment.